dask/tests/test_context.py

```diff
@@ -36,7 +36,7 @@ def bar():
 
 class Foo:
     @globalmethod(key="f")
-    def f():
+    def f():  # type: ignore
```

Comment:
I must admit, I have no idea what the error `mypy` is reporting here is. Seems perfectly fine to me.

Reply:
The ``@globalmethod`` decorator is too magic for mypy to handle.